maxim/logger/llamaindex/__init__.py

```
@@ -0,0 +1,114 @@
+from __future__ import annotations
+
+from typing import Any, Dict, List, Optional, Sequence
+from uuid import uuid4
+
+from llama_index.core.callbacks import BaseCallbackHandler, CBEventType, EventPayload
+from llama_index.core.llms import ChatMessage
+
+from ..logger import GenerationConfig, GenerationError, Logger, TraceConfig
```

Ruff (0.11.9)

`..logger.TraceConfig` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)